frontend/src/components/Inquiries/InquiriesTable.tsx

```diff
@@ -29,38 +31,76 @@ const InquiriesTable = ({ themes }: InquiriesTableProps) => {
   const { data: inquiries } = useInquiries()
 
   const [sortedInquiries, setSortedInquiries] = useState<InquiryPublic[]>([])
+
   useEffect(() => {
-    setSortedInquiries([])
-    if (inquiries?.data) {
-      inquiries?.data.sort((a: InquiryPublic, b: InquiryPublic) => {
-        return (
-          (schedule?.scheduled_inquiries?.indexOf(a.id) ?? -1) -
-          (schedule?.scheduled_inquiries?.indexOf(b.id) ?? -1)
+    if (inquiries) {
+      const inquiriesList = inquiries.data
+        .filter(
+          (i) => (schedule?.scheduled_inquiries?.indexOf(i.id) ?? -1) >= -1,
```

Comment:
I think the conditional in the filter can be simplified with something like
`schedule?.scheduled_inquiries?.includes(i.id)`.